spec.html

```diff
@@ -22565,12 +22581,12 @@ <h2>Syntax</h2>
         `default` `:` StatementList[?Yield, ?Await, ?Return]?
     </emu-grammar>
 
-    <emu-clause id="sec-switch-statement-static-semantics-early-errors">
+    <emu-clause id="sec-switch-statement-static-semantics-early-errors" oldids="sec-switch-duplicates-allowed-static-semantics">
       <h1>Static Semantics: Early Errors</h1>
       <emu-grammar>SwitchStatement : `switch` `(` Expression `)` CaseBlock</emu-grammar>
       <ul>
         <li>
-          It is a Syntax Error if the LexicallyDeclaredNames of |CaseBlock| contains any duplicate entries.
+          It is a Syntax Error if the LexicallyDeclaredNames of |CaseBlock| contains any duplicate entries<span normative-optional>, unless IsStrict(this production) is *false*, the duplicate entries are only bound by FunctionDeclarations, and the host is a web browser or otherwise supports <emu-xref href="#sec-block-level-function-declarations-web-legacy-compatibility-semantics" title></emu-xref></span>.
         </li>
```

Comment:
Likewise here (`<li><p>It is a Syntax Error if…</p><p normative-optional>If the host is a web browser or otherwise supports…</p></li>`).